drake_ros_tf2/test/test_name_conventions.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <string>
+
+#include "../src/utilities/internal_name_conventions.h"
```

Comment:
@aaronchongth consider using `target_include_directories` to add `src` or `src/utilities` as `BUILD_INTERFACE` include path.

Reply:
Good call, that would make the includes much cleaner, thanks! https://github.com/RobotLocomotion/drake-ros/pull/35/commits/a9cf24a09ecb598e648496f411ec301610746ab8